### PR TITLE
Update campaign page

### PR DESCRIPTION
**Pull request by anasofiapinho**
We are closing the campaign for 2016

### CODE REVIEW

**Comment by anasofiapinho** [comment]
@anikalindtner and @mkalininait here is the updated campaign page. Could one of you give me feedback about this or merge if it's ok?

@mkalininait can you change the number of teams to 16 on the bar? I believe we should just leave the amount we funded so for a while and then zero it? How did you do on the other years?

File: campaign.html [hunk 1]
```diff
@@ -25,6 +25,16 @@
 </div>
 
 <div class="margin-top">
+  <p class="stats">
+    <strong>The fundraising campaign for RGSoC 2017 is now open!</strong>
+  </p>
+
+  <p>
+    All the amount we receive from this moment on will be used on next year's edition.
```
Comment (alicetragedy) [inline campaign.html]: Can you change "All the amount" to "All the money"? And "will be used on" -> "will be used for".

File: campaign.html [hunk 2]
```diff
@@ -25,6 +25,16 @@
 </div>
 
 <div class="margin-top">
+  <p class="stats">
```
Comment (alicetragedy) [inline campaign.html]: For consistency's sake, could you remove the class from this paragraph tag? It's not needed.

**Comment by alicetragedy** [comment]
I've left some comments about small changes that need doing; I'm not sure if we should leave the sponsor packages up or not, but if we do you might also want to change "Become a sponsor for 2016" to "Become a sponsor for 2017" ([link](https://github.com/rails-girls-summer-of-code/summer-of-code/pull/229/files#diff-2a4c6593bc7d39e3f92b642b445973b5R88))

**Comment by anasofiapinho** [comment]
@alicetragedy I updated the changes. Let me know if it's ok to merge!

**Pull request merged**